tools/sched_ext/scx_serialise.bpf.c

```diff
@@ -144,59 +183,6 @@ const volatile u32 debug = 1;
 			bpf_printk(fmt, ##args); \
 	} while (0)
 
-/*
- * This flag indicates whether a task has yielded. It is used to signal
- * the dispatch function to dispatch the next highest priority thread.
- */
-// bool yield_flag = false;
-
-/*
- * Insert or update the task context of a task in the task context map. 
- * Call this fn when a task is enqueued, dequeued, or dispatched.
- */
-static int tctx_map_insert(pid_t new_pid, s32 modify_prio, bool enqueued,
```

Comment:
All of the algorithms are still using this function. The current version of the changes don't compile